Source/Orts.Simulation/Simulation/Simulator.cs

```diff
@@ -1955,14 +1955,14 @@ private void StartSwitchPlayerTrain()
                             TrainSwitcher.ClickedSelectedAsPlayer = false;
                             return;
                         }
-                        if (playerTrain.TrainType == Train.TRAINTYPE.AI_PLAYERDRIVEN || !playerTrain.Autopilot)
+                        if (playerTrain.TrainType == Train.TRAINTYPE.AI_PLAYERDRIVEN || TimetableMode && !playerTrain.Autopilot)
                         {
                             // it must be autopiloted first
                             playerTrain.SwitchToAutopilotControl();
                         }
                         // and now switch!
                         playerTrain.TrainType = Train.TRAINTYPE.AI;
-                        playerTrain.Autopilot = false;
```

Comment:
Is removing this intentional? The line is present in ORNYMG

Reply:
Correcting an error and inserting another error.
Thanks!